### PR TITLE
setup_barman testing framework

### PR DESCRIPTION
**Pull request by vincentp7212**
Centos7/Debian10 Error with WAL file

### CODE REVIEW

**Comment by vincentp7212** [comment]
Includes the following tests:

- barman user is successfully created in primary
- barman backup server is configured correctly
- primary server status is active when using barman status function
- barman home_dir is configured correctly

File: tests/tests/test_setup_barman.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    with host.sudo(pg_user):
+        query = "Select * from pg_user where usename = '%s'" % barman_user
+        cmd = host.run('psql -At -p %s -h %s -c "%s" postgres' % (port, socket_dir, query))
+        result = cmd.stdout.strip()
+
+    assert len(result) > 0, \
+        "Barman user does not exist in primary database"
+
+
+
```
Comment (jt-edb) [inline tests/tests/test_setup_barman.py:72]: It would be interesting to check if the `barman backup` command works

**Review by jt-edb — COMMENTED** [review]
Is it possible to rename the folder to `setup_barman`, because there is currently a trailing `_`?
Could you add this to the Github actions?

**Review by jt-edb — APPROVED** [review]
LGTM

**Pull request merged**